<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadiLightWidget</class>
 <widget class="QWidget" name="RadiLightWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1135</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1135</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1135</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_caption">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Radi-RGB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_color_effect">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>45</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Color Effect</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_light">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>70</y>
     <width>252</width>
     <height>570</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>435</y>
     <width>840</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 255)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <widget class="QLabel" name="label_brightness">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Brightness</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_brightness">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>200</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>15</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_speed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>200</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_speed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>67</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_randomcolor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Random Color</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_key_check">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>391</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="checkBox_right_side_led">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>130</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Right side LED</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_select_all">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Select all</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_top_side_led">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>120</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Top side LED</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_wasd">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>WASD</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_number">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>120</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Number row</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_left_side_led">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>120</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Left side LED</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_bottom_side_led">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>145</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Bottom side LED</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_arrows">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Arrows</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_functions">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Functions</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_reset_all_leds">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Reset all LEDs</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox_custom">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_custom">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 225, 0, 255)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
